supplierProduct/tests.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from supplierProduct.models import supplierProduct, Product, Supplier
+
+supID0 = 123
+supID1 = 222
+supID2 = 127
+
+
+class TestSupProduct:
+
+    @pytest.fixture
+    def creat_product(self):
```

Comment:
typo: creat_product -> create_product

but, in the case of a fixture, the name should be the outcome of creating the fixture rather than the action of creating it. e.g. something like `products` instead of `create_product`.